WordPress/src/main/java/org/wordpress/android/ui/compose/components/ImageButton.kt

```diff
@@ -67,18 +72,18 @@ fun ImageButton(
             }
             .clickable { onClick.invoke() }
         ) {
-            val buttonTextValue = when (buttonText) {
-                is UiString.UiStringText -> buttonText.text
-                is UiString.UiStringRes -> stringResource(id = buttonText.stringRes)
-                is UiString.UiStringResWithParams -> stringResource(id = buttonText.stringRes)
+            val buttonTextValue: String = when (button.text) {
```

Comment:
@AjeshRPai I get the feeling you could DRY this by replacing it with `uiStringText(button.text)` or `button.text.asString()` (whichever you prefer) from `org.wordpress.android.ui.compose.utils` 👍🏻

Reply:
@ovitrif Thanks 🙇🏼 . I missed that there is a compose util, I have done the suggested change with the commit [082947b](https://github.com/wordpress-mobile/WordPress-Android/pull/18018/commits/082947b9f9c9191d90a229d7209f9b9c4272a735)